lesson3/Lesson3HW/app/src/main/res/layout/fragment_settings.xml

```diff
@@ -4,30 +4,24 @@
     android:layout_height="match_parent"
     tools:context=".SettingsFragment">
 
-    <!-- TODO: Update blank fragment layout -->
-    <TextView
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:text="@string/hello_blank_fragment" />
-
```

Comment:
This textview was overlapping with your buttons, and it was the default fragment textview that we don't need anyway